sergeant/worker.py

```diff
@@ -615,6 +621,34 @@ def handle_starvation(
                 },
             )
 
+    def handle_stop(
+        self,
+        task: objects.Task,
+    ) -> None:
+        try:
+            if self.config.logging.events.on_stop:
+                self.logger.error(
```

Comment:
A task asking for a stop is not an error. You should change the logging level to either warning or info